docs/source/tutorials/datalab/datalab_advanced.ipynb

```diff
@@ -722,7 +722,7 @@
    "outputs": [],
    "source": [
     "from cleanlab.datalab.issue_manager import IssueManager\n",
-    "from cleanlab.datalab.factory import register\n",
+    "from cleanlab.datalab.issue_manager_factory import register\n",
```

Comment:
please confirm you've properly updated this import in all of the guides too, eg:
https://docs.cleanlab.ai/stable/cleanlab/datalab/guide/custom_issue_manager.html

CC @elisno